include/range/v3/span.hpp

```diff
@@ -97,6 +97,10 @@ namespace ranges
             private:
                 span_index_t size_ = 0;
             };
+
+            template<span_index_t Extent, span_index_t Offset, span_index_t Count>
+            using subspan_extent = std::integral_constant<span_index_t,
+                Count == dynamic_extent && Extent != dynamic_extent ? Extent - Offset : Count>;
```

Comment:
Why isn't this just a `constexpr` function?

Reply:
For absolutely no good reason.